Project_2/djangoProject/cars/forms.py

```diff
@@ -0,0 +1,37 @@
+from django import forms
+from django.core.exceptions import ValidationError
+from django.contrib.auth.forms import UserCreationForm
+from django.contrib.auth.models import User
+
+
+class RegisterUser(UserCreationForm):
+	first_name = forms.CharField(max_length=12, min_length=4, required=True,
+								 widget=forms.TextInput(attrs={'class': 'form-control', 'placeholder': 'John'}))
+	last_name = forms.CharField(max_length=12, min_length=4, required=True,
+								widget=(forms.TextInput(attrs={'class': 'form-control', 'placeholder': 'Doe'})))
+	address = forms.CharField(max_length=12, min_length=4,
+								widget=(forms.TextInput(attrs={'class': 'form-control'})))
+	email = forms.EmailField(max_length=50, widget=(forms.TextInput(attrs={'class': 'form-control',
+																		   'placeholder': 'john.doe@email.com'})))
+	password1 = forms.CharField(label='Password', widget=forms.PasswordInput(attrs={'class': 'form-control'}))
+	password2 = forms.CharField(label='Password confirmation',
+								widget=forms.PasswordInput(attrs={'class': 'form-control'}),
+								help_text='Enter the same password as before, for verification.')
```

Comment:
The code here looks weirdly indented. A better way is to put all the arguments on different lines, or start the indentation on the next line from where the first keyword argument starts (under `max_length`). For example:
```
	first_name = forms.CharField(
            max_length=12, min_length=4, required=True, 
            widget=forms.TextInput(
                attrs={'class': 'form-control', 'placeholder': 'John'}
            )
        )
```

Read this for more detailed information: [PEP-8 indentation](https://peps.python.org/pep-0008/#indentation)